### ISSUE TITLE
How do you split the data

### ISSUE THREAD

**Issue by Layne-Huang** [theislab/IMPA#9]
Hi,

Thanks for your excellent work. I am trying to reproduce your result on bbbc021 dataset. However, I am confused about how you split the training set and test set. For example, you only use one of the cells perturbated by floxuridine and keep others in the training set. Image that the moedl would receive the same input since they use the same drug and similar controlled cells and would try to replicate the cells in the training set. Could you explian about it? Thank you very much!

<img width="996" height="310" alt="Image" src="https://github.com/user-attachments/assets/3ad6c0e4-b425-4351-98a8-c5ffc75e5b76" />

**Comment by allepalma** [comment]
Hi @Layne-Huang,

Thanks for your interest, and sorry for the late answer. The training/test split you see in the data is the one I used to produce fig.1. There, you don't have fully held-out compounds. Experiments on heldout compounds are in fig.2, where the list of compounds I left out of the training process is highlighted in bold.

What results are you trying to reproduce? :)
